### ISSUE TITLE
Contradiction on https://wpfriends.at/follow-me/

### ISSUE THREAD

**Issue by HammyHavoc** [akirk/friends#149]
In the written content on https://wpfriends.at/follow-me/, there's a very confusing contradiction:

"In order to follow that site, you need to have your own blog. If you don’t have one, it’s possible to follow the site using an [RSS Reader](https://alternativeto.net/category/books--news/rss/)."

Evidently, you don't need to have your own blog to follow the site, users can use RSS. There are _options_, and it isn't exclusionary in those options.

**Comment by akirk** [comment]
You are right, thank you! The site is not in a Github repo but [I've updated the text like this](https://wpfriends.at/follow-me/), hopefully to make it more clear:

> If you have your own blog and can install the Friends plugin, you can befriend that page and thus you both become part of the network of blogs.
> 
> If you don't have one, it is possible to follow the site using an [RSS Reader](https://alternativeto.net/category/books--news/rss/).

As I mentioned in #147, that page is probably not the best idea.

**Comment by HammyHavoc** [comment]
Looks good to me! :- )

What might make more sense instead of redirecting people off-site, is a simple on-site message telling users that they don't have the plugin, and if they do still want to send them a friend request, they'll need it, but otherwise can follow by other means as a regular subscription. Unlike other user-facing classes that end up on the Page generated by default, this would need to be more of a conditional error/state message.

My thinking with separating it out from other Page-based text is that this message couldn't be accidentally removed by site owners, but likewise, this message would be possible to update via subsequent updates to the plugin, and not only on first-generation of the Page when installing the plugin.

---

In practice, it should work similarly to an email newsletter subscription box:

1. Enter the email address into the text field
2. Site is polled for Friends plugin being installed
3. If it isn't found, display an on-site error message warning the user of this. Similar to what happens if an email address isn't valid in terms of missing a suffix for example.
